src/go/k8s/internal/controller/redpanda/redpanda_controller.go

```diff
@@ -745,6 +802,11 @@ func (r *RedpandaReconciler) createHelmReleaseFromTemplate(ctx context.Context,
 			Retries:  1,
 			Strategy: &rollBack,
 		},
+		// we skip waiting since relying on the Helm release process
+		// to actually happen means that we block running any sort
+		// of pending upgrades while we are attempting the upgrade job.
+		DisableWait:        true,
+		DisableWaitForJobs: true,
```

Comment:
Heads up that I only disabled the waits on upgrades. We probably could do so too for installs, but 🤷 a bit smaller surface area for now.

Reply:
That feels reasonable to me!